### PR TITLE
Display lifecycle badges correctly

### PR DESCRIPTION
**Pull request by yutannihilation**
Currently, preformatted texts are displayed. I guess this is a mistake of copying and pasting the example in `compat-lifecycle.R`. This PR removes ```` ``` ```` and replaces `mypkg` with `dplyr`.

Besides, the filename of images are changed at the time of this commit: https://github.com/tidyverse/dplyr/commit/cb078caf26906e5b542fd16e626b90b538dc5428, so I copied them from rlang's repo.

### CODE REVIEW

**Comment by yutannihilation** [comment]
I'm really not sure why Travis attempt to build the PDF manual... Does adding `\Sexpr{}` macro invoke the build?

**Comment by yutannihilation** [comment]
Ah, according to the help of `R CMD build`, it seems `\Sexprs{}` matters...

```
https://github.com/tidyverse/dplyr/blob/c8dafbad4e94d52e8efcc63b4476eb8ca930ad16/.travis.yml#L7
```

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ dist: trusty
 cache: packages
 latex: false
 fortran: false
+r_build_args: '--no-manual'
```
Comment (romainfrancois) [inline .travis.yml]: is that ok @hadley ?

File: R/bind.r [hunk 2]
```diff
@@ -132,9 +132,7 @@ bind_cols <- function(...) {
 
 #' Combine vectors
 #'
-#' ```
-#' \Sexpr[results=rd, stage=render]{mypkg:::lifecycle("questioning")}
-#' ```
+#' \Sexpr[results=rd, stage=render]{dplyr:::lifecycle("questioning")}
```
Comment (romainfrancois) [inline R/bind.r]: 🤦‍♂️ OMG this is embarrassing. Thanks.

**Pull request merged**